open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/gaian/GaianQueryConstructor.java

```diff
@@ -13,12 +13,7 @@
 import org.springframework.beans.factory.annotation.Value;
```

Comment:
Should not have Gaian specific code in the virtualizer

Reply:
Hi Mandy, the code in the folder is the old version of virtualizer. It will be deprecated and removed from the repo when the OMAG server is done. :)